tinnsleep/visualization.py

```diff
@@ -2,35 +2,88 @@
 import numpy as np
 
 
-def create_visual(raw, detect, duration, interval):
-    """ Create two visuals of a burst detection : a simple plot enabling 
-    visualization of time repartition and a display over the raw signal as annotations.
-    Note: any existing annotation of the raw instance will be removed.
-    
+def plotTimeSeries(data,
+                   ch_names=None,
+                   sfreq=1,
+                   scalings=None,
+                   annotations=None,
+                   ax=None,
+                   **kwargs):
+    """Advanced plotting of multidimensional time series from ndarray
+
     Parameters
     ----------
-    raw : instance of mne.Raw
-        the signal of interest
-    detect: ndarray, shape (n_trials,)
-        the vector of labels for the annotation sampled at the interval rate
-    duration: int
-        Number of elements (i.e. samples) on the epoch.
-    interval: int
-        Number of elements (i.e. samples) to move for the next epoch (if duration == interval, no overlap)
-    
+    data: array-line, shape (n_samples, n_dimension)
+        multidimensional time series
+    ch_names: list | iterable, shape (n_dimension,) | None
+        the labels for the time series
+    sfreq: float (default: 1)
+        sample rate (in Hz)
+    scalings: float | iterable, shape (n_dimension,)
+        value between two channels
+    annotations: a instance mne.Annotations | list of dictionary (default: {})
+        a list of annotation or dictionary containing the following fields:
+        {'onset': float (seconds), 'duration': float (seconds), 'description': str, orig_time': float (seconds)}
+        Example:
+        >>> # a list of one annotation starting after 0.5 second of duration 1.0 second named 'blink'
+        >>> annotations = [{'onset': 0.5, 'duration': 1.0, 'description': "blink", 'origin_time': 0.0}]
```

Comment:
outdated: removed parameters

I think that whenever possible we should use duration and interval in seconds. 
However it may be overkilled in some very basic functions (e.g. `epochs`). 

Read #25 to continue this discussion